src/backend_model.cc

```diff
@@ -330,6 +330,9 @@ TritonModel::UpdateModelConfig(
   auto outputs_config = config.mutable_output();
   *outputs_config = updated_config.output();
 
+  auto dynamic_batching_config = config.mutable_dynamic_batching();
```

Comment:
Need to be careful here, what if `sequence_batching` has been set for the model from the beginning, I think this will clear that because scheduler choice is "oneof" field. We probably should fail the config update, and on the backend side, the auto-complete should check if scheduler choice has been set and don't overwrite it.

Reply:
Updated the method to have the ability to update all batching types when the current config is different from the updated config.